test/Microsoft.NET.Publish.Tests/Microsoft.NET.Publish.Tests.csproj

```diff
@@ -32,6 +32,8 @@
     </PackageReference>
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="$(RepositoryRootDirectory)src\Tasks\Microsoft.NET.Build.Tasks\PackageInfo.cs" Exclude="$(GlobalExclude)" />
```

Comment:
The logic is used to verify the output of the integration test. I can duplicate the logic, but @dsplaisted was ok with using this to build test.